src/app/qgisapp.cpp

```diff
@@ -7195,7 +7195,7 @@ void QgisApp::dxfExport()
     dxfExport.setDestinationCrs( d.crs() );
     dxfExport.setForce2d( d.force2d() );
 
-    QgsDxfExport::Flags flags = nullptr;
+    QgsDxfExport::Flags flags = QgsDxfExport::Flags();
```

Comment:
I guess this is redundant now, and 

```suggestion
    QgsDxfExport::Flags flags;
```

would have the same meaning?